apps/prairielearn/src/schemas/schemas/infoAssessment.json

```
@@ -371,7 +387,8 @@
           "default": 1
         },
         "advanceScorePerc": {
-          "$ref": "#/definitions/advanceScorePerc"
+          "description": "Minimum score percentage to unlock access to subsequent questions",
```

Let's change the corresponding Zod schema:

```suggestion
          "description": "Minimum score percentage to unlock access to subsequent questions.",
```

---

Done in d41984e.